docs/build/build-open-source.md

```diff
@@ -151,7 +151,7 @@ In the below sections you can find a list of different layers of the Polkadot St
+| Gaming                          | [Bit.country](https://github.com/bit-country/Bit-Country-Blockchain) 🟢, [SubGame](https://github.com/SubGame-Network/subgame-network) 🔴, [subzero](https://github.com/playzero/subzero) 🔴, [Web3Games](https://github.com/web3gamesofficial/web3games-blockchain) 🟡, [Ajuna Network](https://github.com/ajuna-network/Ajuna) 🟢, [Gafi Network](https://github.com/cryptoviet/gafi) 🔴, [Asylum](https://gitlab.com/asylum-space/asylum-item-nft) 🟢, [Polket](https://github.com/polketio) 🟢                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
```

Comment:
Probably not a good idea to include just the GitHub profile. Is https://github.com/polketio/polket-node the correct repo to track?

```suggestion
| Gaming                          | [Bit.country](https://github.com/bit-country/Bit-Country-Blockchain) 🟢, [SubGame](https://github.com/SubGame-Network/subgame-network) 🔴, [subzero](https://github.com/playzero/subzero) 🔴, [Web3Games](https://github.com/web3gamesofficial/web3games-blockchain) 🟡, [Ajuna Network](https://github.com/ajuna-network/Ajuna) 🟢, [Gafi Network](https://github.com/cryptoviet/gafi) 🔴, [Asylum](https://gitlab.com/asylum-space/asylum-item-nft) 🟢, [Polket](https://github.com/polketio/polket-node) 🟢                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
```

Reply:
Yea it's correct, I put it because they are working in two different repos (that one, which has the chain logic, and [this](https://github.com/polketio/toearnfun_flutter_app) one, which is the mobile app). Also there are a lot of projects that work with multiple repos.

Reply:
Sounds good to me!

Reply:
done!